openwisp_notifications/tests/test_notifications.py

```diff
@@ -293,9 +295,12 @@ def test_notification_type_message_template(self):
             self.assertEqual(n.email_subject, '[example.com] Messsage Template Subject')
 
         with self.subTest('Links in notification message'):
+            exp_url = _get_absolute_url(
```

Comment:
what does exp stand for? Why not just `url`?
Applies to other occurrences.

Reply:
"exp" = "Expected" 😅

Reply:
let's just use `url`